.github/workflows/codeql.yml

```diff
@@ -0,0 +1,93 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+    - main
+    - release-1[1-9].[0-9]
+    - codeql-workflow # TODO: remove this after PR is complete
+  schedule:
+    - cron: '59 23 * * *'
```

Comment:
Note: Runs triggered by `on.schedule` checks out the head of the default branch at `actions/checkout` step unless it is explicitly set.